src/transport/test.rs

```diff
@@ -290,13 +290,22 @@ fn setup_default_handlers(handlers: &mut Vec<TestHandler>) {
         TestHandler::new("/redirect-to", |_uri, _req, w| {
             write!(
                 w,
-                "HTTP/1.1 302 OK\r\n\
+                "HTTP/1.1 302 FOUND\r\n\
                 Location: /get\r\n\
                 Content-Length: 22\r\n\
                 \r\n\
                 You've been redirected\
                 ",
-            )
+            )?;
+            write!(
+                w,
+                "HTTP/1.1 200 OK\r\n\
+                Content-Type: application/json\r\n\
+                Content-Length: {}\r\n\
+                \r\n",
+                HTTPBIN_GET.as_bytes().len()
+            )?;
+            w.write_all(HTTPBIN_GET.as_bytes())
```

Comment:
I don't think this will really do anything since ureq will make a new request to `/get` as per the `Location` header of the redirect.

Reply:
I can't imagine it working. There will be a new mpsc channel for the next request, so writing the second response here should just be discarded.